web/src/utils/factory-management/buildings.ts

```diff
@@ -21,13 +21,13 @@ export const calculateBuildingRequirements = (factory: Factory, gameData: DataIn
       }
 
       const buildingData = recipe.building
-      const buildingCount = (product.amount / productInRecipe.perMin).toFixed(3)
+      const buildingCount = Number((product.amount / productInRecipe.perMin).toFixed(3))
 
       product.buildingRequirements = {
         name: buildingData.name,
         amount: buildingCount,
-        powerPerBuilding: Number(buildingData.power),
-        totalPower: (parseInt(buildingData.power) * parseInt(buildingCount)).toFixed(2),
+        powerPerBuilding: buildingData.power,
+        totalPower:  Number((buildingData.power * buildingCount).toFixed(3)),
```

Comment:
So is the intention to store totalPower as a string / float? This is the primary reason why I had them as strings in the first place as I couldn't readily use Number type with `toFixed`.

Reply:
Ah never mind, it's wrapped in Number, now why didn't I think of that 😂